diff -uNr a/package/bzip2/bzip2.mk b/package/bzip2/bzip2.mk
--- a/package/bzip2/bzip2.mk	2014-02-27 21:51:23.000000000 +0100
+++ b/package/bzip2/bzip2.mk	2018-09-24 16:03:14.480558086 +0200
@@ -5,7 +5,8 @@
 ################################################################################
 
 BZIP2_VERSION = 1.0.6
-BZIP2_SITE = http://www.bzip.org/$(BZIP2_VERSION)
+#BZIP2_SITE = http://www.bzip.org/$(BZIP2_VERSION)
+BZIP2_SITE = https://fossies.org/linux/misc
 BZIP2_INSTALL_STAGING = YES
 BZIP2_LICENSE = bzip2 license
 BZIP2_LICENSE_FILES = LICENSE
